web-app/js/portal/service/CatalogSearcher.js

```diff
@@ -212,6 +212,21 @@ Portal.service.CatalogSearcher = Ext.extend(Ext.util.Observable, {
         return deepestFacets;
     },
 
+    _brokenMode: function(url) {
+        var getParams = url.split("#")[0].split("?");
```

Comment:
I don't think you should `urlDecode` before splitting the `#` and `?`. I took some code from:
https://github.com/aodn/aodn-portal/blob/master/web-app/js/portal/data/MetadataRecordFetcher.js#L54